static/templates/settings/account_settings.hbs

```diff
@@ -159,7 +159,8 @@
                       {{#unless page_params.is_admin}}{{#if page_params.realm_avatar_changes_disabled}}style="display:none"{{else}}{{#if page_params.server_avatar_changes_disabled}}style="display:none"{{/if}}{{/if}}{{/unless}}>
                         &times;
                     </span>
-                    <div id="user-avatar-source">
+                    <div id="user-avatar-source"
+                      {{#unless page_params.is_admin}}{{#if page_params.realm_avatar_changes_disabled}}class="avatar-change-disabled"{{else}}{{#if page-params.server_avatar_changes_disabled}}class="avatar-change-disabled"{{/if}}{{/if}}{{/unless}}>
```

Comment:
Instead of using overcomplicated Handlebars logic, can you rewrite this into JS similar to how Tim did in 838fcfa83943?

Reply:
Okay, I can do that.